### PR TITLE
Showing questions

### PR DESCRIPTION
**Pull request by Kot-Anna**
Updates:
Question class
showQue servlet
admin-questions.jsp

### CODE REVIEW

File: .classpath [hunk 1]
```diff
@@ -23,6 +23,6 @@
 			<attribute name="org.eclipse.jst.component.dependency" value="/WEB-INF/lib"/>
 		</attributes>
 	</classpathentry>
-	<classpathentry kind="lib" path="C:/Users/kota/eclipse-workspace/Exercise2.3 - resultset/lib/mysql-connector-java-5.1.19-bin.jar"/>
+	<classpathentry kind="lib" path="src/main/webapp/WEB-INF/lib/mysql-connector-java-5.1.19-bin.jar"/>
```
Comment (rhexa) [inline .classpath:26]: this is the driver for MySQL 5.7

**Review by MrConnorr — APPROVED** [review]
We will need to change some names to make everything working

File: src/main/java/app/ShowQue.java [hunk 2]
```diff
@@ -1,19 +1,30 @@
...
-@WebServlet("/ShowQue")
+@WebServlet("/jsp/admin-questions")
 public class ShowQue extends HttpServlet {
 	private static final long serialVersionUID = 1L;
-       
+	private Dao dao=null;
+	
+	@Override
+	public void init() {
+		dao = new Dao("jdbc:mysql://localhost:3306/vaalikone", "root", "Password1");
```
Comment (rhexa) [inline src/main/java/app/ShowQue.java:26]: Just a silly reminder to use your own db config

**Review by rhexa — APPROVED** [review]
Well done, keep the good work!

**Pull request merged**